vision_agent_tools/tools/text_to_object_detection.py

```diff
@@ -44,6 +50,11 @@ def __call__(
         """
         results = []
 
+        if image is None and video is None:
+            raise ValueError("Either 'image' or 'video' must be provided.")
+        if image is not None and video is not None:
+            raise ValueError("Only one of 'image' or 'video' can be provided.")
+
         for prompt in prompts:
             prediction = self.model(image=image, task=prompt)
             output = TextToObjectDetectionOutput(output=prediction)
```

Comment:
this is not correct also. we should not do a for loop through the prompts but pass all the prompts to the owlv2 model.

Reply:
this one is on me, I am fixing it on my new PR:
https://github.com/landing-ai/vision-agent-tools/pull/42/files#diff-1e615d53dc76cd22cc3bf26063206a03d1f3f0c8e21d6770910cbe84648759ab